extension-androidauto/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+  id("com.android.library")
+  kotlin("android")
+  id("org.jetbrains.dokka")
+}
+
+android {
+  compileSdkVersion(AndroidVersions.compileSdkVersion)
+  defaultConfig {
+    minSdkVersion(AndroidVersions.minAndroidAutoSdkVersion)
+    targetSdkVersion(AndroidVersions.targetSdkVersion)
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+  }
+
+  testOptions {
+    unitTests.apply {
+      isIncludeAndroidResources = true
+    }
+  }
+}
+
+dependencies {
+  implementation(project(":sdk"))
+  implementation(Dependencies.googleCarAppLibrary)
+  implementation(Dependencies.kotlin)
+  implementation(Dependencies.androidxCoreKtx)
+  implementation(Dependencies.androidxAnnotations)
+  testImplementation(Dependencies.junit)
+  testImplementation(Dependencies.mockk)
+  testImplementation(Dependencies.androidxTestCore)
+  testImplementation(Dependencies.robolectric)
+  testImplementation(project(":plugin-animation"))
```

Comment:
Is it needed?

Reply:
no, will remove. But would like @kiryldz's input here since we are doing the gestures handling in this extension, will it make sense to use animation plugin's method to reflect the owner of the camera change?

Reply:
it's not needed here because as far as I see we do not have tests, so no need for `testImplementation` for now